src/main/java/org/broadinstitute/consent/http/service/UserService.java

```diff
@@ -208,13 +208,7 @@ public List<User> getUsersAsRole(User user, String roleName) {
             case Resource.SIGNINGOFFICIAL :
                 Integer institutionId = user.getInstitutionId();
                 if (Objects.nonNull(user.getInstitutionId())) {
-                    List<User> institutionUsers = userDAO.getUsersFromInstitutionWithCards(institutionId);
-                    List<User> unregisteredUsers = userDAO.getCardsForUnregisteredUsers(institutionId);
-                    return Stream.of(
-                        institutionUsers,
-                        unregisteredUsers
-                    ).flatMap(Collection::stream)
-                    .collect(Collectors.toList());
+                    return userDAO.getUsersFromInstitutionWithCards(institutionId);
```

Comment:
I should have caught this before. This will only find users in the institution WITH LCs. I'm pretty sure we want users in the institution. I think we might need to revisit this entire code path to ensure we're getting the right thing.

Reply:
on the query itself:
>            //This will pull in users tied to the institution
>            //Users will come with LCs issued by SOs institution (if any)
I think the `WithCards` is referring to the fact that if the user has any library cards it is populated when returned by the db, but it does not filter to only users with library cards.

Reply:
Ok, thanks, false alarm - love it when I'm wrong.

Reply:
When I test by looking at the UI,  researchers without cards in my local environment appear in the table.

Reply:
I still think we need `List<User> unregisteredUsers = userDAO.getCardsForUnregisteredUsers(institutionId);`. That finds all entries where a SO has created a "User" in their institution.

Reply:
>> UI PR: https://github.com/DataBiosphere/duos-ui/pull/1737

> Removes the ability to add unregistered users by removing the 'Add New Researcher' modal.

Ah, never mind, I missed that we were removing that feature.